content/docs/rfcs/23/README.md

```diff
@@ -0,0 +1,155 @@
+
+### Separation of two applications example
+
+Let's say we have some M2M topic that is heavy and experience a lot of heavy traffic and a Status Network one. This can be segregated into:
+
+```
+/waku/2/waku-status/proto
+/waku/2/waku-walletconnect/proto
+```
+
+This indicates that they are WakuMessages but for different domains completely.
+
+### Topic sharding example
+
+Topic sharding is currently not supported by default, but is planned for the future in order to deal with increased network traffic. Here's a sketch for what this might look like:
+
+```
+waku/2/waku-9_shard-0/proto
+...
+waku/2/waku-9_shard-9/proto
+```
+
+This indicates explicitly that the network traffic has been partitioned into 10 buckets.
+
+### Compression example
+
+Not yet implemented, but would be easy to add with:
+
+`/waku/2/default-waku/proto_snappy`
+
+### Why use a single PubSub topic by default?
+
+Using a single PubSub topic ensures a connected network, as well some degree of metadata protection. See [section on Anonymity/Unlinkability](/spec/10/#anonymity--unlinkability).
+
+If you use another PubSub topic, be aware that metadata protection might be weakened,
+and other nodes in the network might not subscribe or store messages for your given Pubsub topic.
+This means you are likely to have to run your own full nodes which may make your application less robust.
+
+## Content topics
+
+The other type of topic that exists in Waku v2 is content topics.
+This is used for content based filtering.
+See [14/WAKU2-MESSAGE spec](/spec/14) for where this is specified.
+Note that this doesn't impact routing of messages between relaying nodes,
+but it does impact how request/reply protocols such as 
+[12/WAKU2-FILTER](https://rfc.vac.dev/spec/12/) and [13/WAKU2-STORE](https://rfc.vac.dev/spec/13/) are used.
+
+This is especially useful for nodes that have limited bandwidth,
+and only want to pull down messages that match this filter.
+
+Since all messages are relayed regardless of content topic, you MAY use any content topic you wish without impacting how messages are relayed.
+
+### Content topic format
+
+The format of content topics is as follows:
+
+`/waku/2/ContentTopic/Encoding`
+
+The name of ContentTopic is application-specific. As an example, here's the content topic used for an upcoming testnet:
+
+`/waku/2/huilong/proto`
+
+## Differences with Waku v1
+
+In [6/WAKU1](/spec/6) there is no actual routing.
+All messages are sent to all other nodes.
+This means that we are implicitly using the same PubSub topic that would be something like:
+
+```
+/waku/1/default-waku/rlp
+```
+
+Topics in Waku v1 correspond to Content Topics in Waku v2.
+
+### Bridging Waku v1 and Waku v2
+
+To bridge Waku v1 and Waku v2 we have a [15/WAKU-BRIDGE](/spec/15).
+For mapping Waku v1 topics to Waku v2 content topics,
+the following structure is used:
+
+```
+/waku/v1/<4bytes-waku-v1-topic>/rlp
```

Comment:
Thanks for specifying this. Not yet implemented as such for bridge, but see https://github.com/status-im/nim-waku/issues/576.

Curious why the encoding here is set to `rlp`, as the v1 payload would have been encapsulated in a proto-encoded v2 message? I guess it's a matter of interpretation.

```suggestion
/waku/1/<4bytes-waku-v1-topic>/rlp
```

Reply:
Ah good point, /proto would probably be more accurate here